code/modules/admin/verbs/ert.dm

```diff
@@ -109,36 +111,73 @@
 		ertemplate.teamsize = prefs["teamsize"]["value"]
 		ertemplate.mission = prefs["mission"]["value"]
 		ertemplate.polldesc = prefs["polldesc"]["value"]
-		ertemplate.enforce_human = prefs["enforce_human"]["value"] == "Yes" // these next 5 are effectively toggles
+		ertemplate.enforce_human = prefs["enforce_human"]["value"] == "Yes" // these next 6 are effectively toggles
 		ertemplate.opendoors = prefs["open_armory"]["value"] == "Yes"
 		ertemplate.leader_experience = prefs["leader_experience"]["value"] == "Yes"
 		ertemplate.random_names = prefs["random_names"]["value"] == "Yes"
 		ertemplate.spawn_admin = prefs["spawn_admin"]["value"] == "Yes"
+		ertemplate.use_custom_shuttle = prefs["use_custom_shuttle"]["value"] == "Yes"
 
 		var/list/spawnpoints = GLOB.emergencyresponseteamspawn
 		var/index = 0
 
+		var/list/mob/dead/observer/candidates = poll_ghost_candidates("Do you wish to be considered for [ertemplate.polldesc]?", "deathsquad")
+		var/teamSpawned = FALSE
+
+		// This list will take priority over spawnpoints if not empty
+		var/list/spawn_turfs = list()
+
+		// Takes precedence over spawnpoints[1] if not null
+		var/turf/brief_spawn
+
+		if(!length(candidates))
+			return FALSE
+
+		if(ertemplate.use_custom_shuttle && ertemplate.ert_template)
+			to_chat(usr, span_boldnotice("Attempting to spawn ERT custom shuttle, this may take a few seconds..."))
+			var/datum/map_template/shuttle/ship = new ertemplate.ert_template
+			var/x = rand(TRANSITIONEDGE, world.maxx - TRANSITIONEDGE - ship.width)
+			var/y = rand(TRANSITIONEDGE, world.maxy - TRANSITIONEDGE - ship.height)
+			var/z = SSmapping.empty_space.z_value
+			var/turf/located_turf = locate(x, y, z)
+			if(!located_turf)
+				CRASH("ERT shuttle found no place to load in")
+
+			if(!ship.load(located_turf))
+				CRASH("Loading ERT shuttle failed!")
+
+			var/list/shuttle_turfs = ship.get_affected_turfs(located_turf)
+
+			for(var/turf/affected_turf as anything in shuttle_turfs)
+				for(var/obj/effect/landmark/ert_shuttle_spawn/spawner in affected_turf)
+					spawn_turfs += get_turf(spawner)
+
+				if(!brief_spawn)
+					brief_spawn = locate(/obj/effect/landmark/ert_shuttle_brief_spawn) in affected_turf
+
+			if(!length(spawn_turfs))
+				stack_trace("ERT shuttle loaded but found no spawnpoints, placing the ERT at wherever inside the shuttle instead.")
+
+				for(var/turf/open/floor/open_turf in shuttle_turfs)
```

Comment:
I get that this is unwanted behavior, but if you're going to do a fallback to "anywhere on the shuttle that might work", maybe do an `is_safe_turf` check so you don't spawn them inside an engine surrounded by walls/space.

Reply:
Done